packages/cosmos/src/client.ts

```diff
@@ -420,6 +452,24 @@ function toBodyInit(obj: DocumentInit): BodyInit {
   return JSON.stringify(obj);
 }
 
+type PartitionKeyRanges = {
+  _rid: string;
+  PartitionKeyRanges: {
+    _rid: string;
+    id: string;
+    _etag: string;
+    minInclusive: string;
+    maxExclusive: string;
+    ridPrefix: number;
+    _self: string;
+    throughputFraction: number;
+    status: string;
+    parents: unknown[];
+    _ts: number;
+  }[];
+  _count: number;
+};
```

Comment:
would be good to move this to types.ts and create a separate interface for PartitionKeyRange that extends the persisted object interface.

Reply:
See latest comment